### PR TITLE
hotpot connector for NFS

### PR DESCRIPTION
**Pull request by jackhaibo**
**What this PR does / why we need it**:
Connector of file share `NFS` protocol used by sushi.

### CODE REVIEW

**Comment by leonwanghui** [comment]
@wisererik @xxwjj PTAL

File: contrib/connector/connector.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ const (
 
 	NvmeofDriver = config.NVMEOFProtocol
 	Nqn          = "nqn"
+	NFSDriver    = "NFS"
```
Comment (leonwanghui) [inline contrib/connector/connector.go]: Please use config.NFSProtocol instead.

File: contrib/connector/nfs/helper.go [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+	"github.com/mitchellh/mapstructure"
+	"github.com/opensds/opensds/contrib/connector"
+)
+
+import (
+	"fmt"
+)
+
+type NFSConnectorInfo struct {
+	ExportLocations []string `mapstructure:"ExportLocations"`
```
Comment (leonwanghui) [inline contrib/connector/nfs/helper.go]: is here ExportLocations or exportLocations?

File: contrib/connector/connector.go (hunk 1 above)
Comment (jackhaibo) [reply contrib/connector/connector.go]: Done.

File: contrib/connector/nfs/helper.go (hunk 2 above)
Comment (jackhaibo) [reply contrib/connector/nfs/helper.go]: Change it to exportLocations.

**Pull request merged**